package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "edx-ora2",
-  "version": "5.0.4",
+  "version": "5.0.5",
```

[ditto](https://github.com/openedx/edx-ora2/pull/1928/files#diff-60d133f8459119b2c44feadd371cc76e9d88161518789eefc8173b2d60197cb6R4):
```suggestion
  "version": "5.1.1",
```